cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -334,6 +367,11 @@ private void update(Network network, Context updateContext, ReportNode reportNod
 
         network.runValidationChecks(false, reportNode);
         network.setMinimumAcceptableValidationLevel(ValidationLevel.STEADY_STATE_HYPOTHESIS);
+
+        // Remove all properties and aliases, this will invalidate all subsequent updates
+        if (updateContext.config().getRemovePropertiesAndAliasesAfterImport()) {
+            removeAllAliasesAndProperties(network);
+        }
```

Comment:
The documentation states that:

> When this option is set to true, all generated properties and aliases are removed after the first import.

Due to its location, this operation is also performed after each update. So you can import the EQ profile alone, and then update the network with another profiles, and remove the properties and aliases by providing the import parameter only for the update operation.
I think that this behavior can be useful. But if we keep it, the documentation should be changed.